### PR TITLE
Avg and sum of Durations

### PR DESCRIPTION
**Pull request by sherfert**
Corresponding product PR: https://github.com/neo4j/neo4j/pull/11808

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AggregatingFunctionsTest.scala [hunk 1]
```diff
@@ -281,7 +295,7 @@ class AggregatingFunctionsTest extends DocumentingTest {
         resultTable()
       }
     }
-    section("sum()", "functions-sum") {
+    section("sum() - Numeric values", "functions-sum") {
```
Comment (Lojjs) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AggregatingFunctionsTest.scala:298]: You forgot to do this change to the old avg section

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AggregatingFunctionsTest.scala [hunk 2]
```diff
@@ -96,6 +98,18 @@ class AggregatingFunctionsTest extends DocumentingTest {
         resultTable()
       }
     }
+    section("avg() - Durations", "functions-avg-duration") {
+      p(
+        "`avg()` returns the average of a set of Durations.")
+      function("avg(expression)", "A Duration.", ("expression", "An expression returning a set of durations."))
+      considerations("Any `null` values are excluded from the calculation.", "`avg(null)` returns `null`.")
+      query("UNWIND [duration('P2DT3H'), duration('PT1H45S')] AS dur RETURN avg(dur)", ResultAssertions(r => {
+        r.toList.head("avg(dur)") should equal(DurationValue.duration(0, 1, 2 * 60 + 22, 500000000))
```
Comment (Lojjs) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AggregatingFunctionsTest.scala]: Shouldn't it be `2 * 3600 + 22` seconds?

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AggregatingFunctionsTest.scala [hunk 3]
```diff
@@ -292,6 +306,17 @@ class AggregatingFunctionsTest extends DocumentingTest {
         resultTable()
       }
     }
+    section("sum() - Durations", "functions-sum-duration") {
+      p("`sum()` returns the sum of a set of durations.")
+      function("sum(expression)", "A duration.", ("expression", "An expression returning a set of durations."))
+      considerations("Any `null` values are excluded from the calculation.")
+      query("UNWIND [duration('P2DT3H'), duration('PT1H45S')] AS dur RETURN sum(dur)", ResultAssertions(r => {
+        r.toList.head("sum(dur)") should equal(DurationValue.duration(0, 3, 4 * 60 + 45, 0))
```
Comment (Lojjs) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AggregatingFunctionsTest.scala]: Shouldn't it be `DurationValue.duration(0, 2, 4 * 3600 + 45, 0)` ?

**Review by Lojjs — CHANGES_REQUESTED** [review]
You have done some calculation errors in your examples and missed to add `- Numeric values` in one place. You are also mixing btween `duartions` and `Durations`, e.g. `returns the average of a set of Durations` but `returns the sum of a set of durations`

**Review by Lojjs — APPROVED** [review]

**Review by jjaderberg — APPROVED** [review]
LGTM

**Pull request merged**